### PR TITLE
feat: add Reactions

### LINKED ISSUE #65: add reactions

### PR DESCRIPTION
**Pull request by spaenleh**
This PR adds reactions to the Code Review App.

Available reactions at this time are:
- 👍  +1
- 👎  -1
- 😄  laugh
- ❤️  heart
- 🚀  rocket

Some screenshots:

Capture d’écran 2022-02-23 à 16 40 15
Capture d’écran 2022-02-23 à 16 40 29

closes #65

### CODE REVIEW

**Comment by spaenleh** [comment]
@juancarlosfarah There is a "bug" (or is it a feature ?) where the user can add the same reaction multiple times using the "dropdown" list. If we want to replicate Github exactly, we should highlight any reaction in the list that the user has already pressed and pressing it again should delete it.

Otherwise pressing the displayed reactions (little chips) does the job (i.e. adding when "deselected" or removing when already given by the user).

**Review by juancarlosfarah — APPROVED** [review]
👍

**Comment by spaenleh** [comment]
UPDATE: as of 64a8520 the emojis in the picker are the ones from GitHub + 😂

Capture d’écran 2022-02-24 à 14 25 26
Capture d’écran 2022-02-24 à 14 25 55

Overall small changes, @juancarlosfarah if you want to have a look to make sure, otherwise i will be merging it.

File: src/components/common/CommentView.js [hunk 1]
```diff
@@ -411,9 +514,104 @@ class CommentView extends Component {
...
+              (row) => (
+                <Grid container item direction="row">
+                  {row.map((reaction) => (
+                    <Grid item>
+                      <Button
+                        className={classes.emojiButton}
+                        key={reaction.label}
+                        disableElevation
+                        color="primary"
+                        // here we use undefined because when using "filled" it does not work
```
Comment (juancarlosfarah) [inline src/components/common/CommentView.js:588]: We can remove this comment.

File: src/components/common/CommentView.js [hunk 2]
```diff
@@ -411,9 +514,104 @@ class CommentView extends Component {
...
+      <Grid container item direction="row" spacing={1} alignItems="center">
+        <Grid item>{this.renderAddReactions('small')}</Grid>
+        {reactions.map((reaction) =>
+          reaction.count ? (
+            <Grid item key={reaction.label}>
+              <Chip
+                label={`${reaction.icon} ${reaction.count}`}
+                size="small"
+                color="primary"
+                // here we use undefined because when using "filled" it does not work
```
Comment (juancarlosfarah) [inline src/components/common/CommentView.js:533]: Confirm that you can use `contained`.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looks good! Just some minor changes.

File: src/components/common/CommentView.js (hunk 2 above)
Comment (spaenleh) [reply src/components/common/CommentView.js:533]: In MUI v4, the variant property is called `'default'` to get the "filled" appearance.

**Review by juancarlosfarah — APPROVED** [review]
👍

**Pull request merged**